src/config.rs

```diff
@@ -64,4 +69,24 @@ impl Config {
             self.paste.filename = args.filename.as_ref().cloned();
         }
     }
+
+    /// Parses files referenced by [Config::auth_token_file] and [Config::delete_token_file] and
+    /// moves them into their respective token variable.
+    pub fn parse_token_files(&mut self) {
```

Comment:
We should handle tokens line by line similarly to the server and also document their usage.

As of now it is not clear what the contents of `*_token_file` should be.

Reply:
And with #66 implemented it would only make sense to allow 1 auth & deletion token per profile

Or are you talking about allowing to set auth and deletion token from one file instead of having to pass 2?

Reply:
Hm yeah you're right, I was confusing this functionality with the server a bit. I think adding a comment/documentation about the contents of the file would be sufficient in this case.